robocop/checkers/naming.py

```diff
@@ -676,55 +677,51 @@ def find_similar_variables(self, tokens, node):
 class DeprecatedStatementChecker(VisitorChecker):
     """Checker for deprecated statements."""
 
-    reports = (
-        "deprecated-statement",
-    )
-    # deprecated:alternative
-    deprecated_statements_rf3 = {}
-    deprecated_statements_rf4 = {"runkeywordunless": "IF"}
-    deprecated_statements_rf5 = {"runkeywordunless": "IF",
-                                 "runkeywordif":     "IF"
-                                 }
-
-    def visit_SuiteSetup(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
-
-    def visit_TestSetup(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
-
-    def visit_Setup(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
-
-    def visit_SuiteTeardown(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
+    reports = ("deprecated-statement",)
+    depreccated_st = {5: {"[Return]"}}
```

Comment:
I would put a longer and more descriptive name for this variable. (also there was a typo)
```suggestion
    deprecated_statements = {5: {"[Return]"}}
```